doc/sphinxext/mne_substitutions.py

```diff
@@ -34,6 +35,27 @@ def run(self, **kwargs):  # noqa: D102
                 f"plot in *{DEFAULTS['units'][key]}*)"
                 for key in keys
             )
+        elif self.arguments[0] == "non-data channels list":
+            keys = list()
+            rst = ""
+            for key in _DATA_CH_TYPES_ORDER_DEFAULT:
+                if (
+                    not _PICK_TYPES_DATA_DICT.get(key, True)
+                    or key in _EYETRACK_CH_TYPES_SPLIT
+                    or key in ("ref_meg", "whitened")
+                ):
+                    keys.append(key)
+            for key in keys:
+                if DEFAULTS["scalings"].get(key, False) and DEFAULTS["units"].get(
+                    key, False
+                ):
+                    rst += (
+                        f"- ``{repr(key)}``: **{DEFAULTS['titles'][key]}** "
+                        f"(scaled by {DEFAULTS['scalings'][key]} to "
```

Comment:
Can you improve the formatting here? It results in:
```
'mag': Magnetometers (scaled by 1000000000000000.0 to plot in fT)
```
perhaps
```suggestion
                        f"(scaled by {DEFAULTS['scalings'][key]:g} to "
```

Reply:
Right, I will add it for the 'data channels' too